sherpa-onnx/csrc/session.cc

```diff
@@ -53,6 +64,57 @@ static Ort::SessionOptions GetSessionOptionsImpl(int32_t num_threads,
       }
       break;
     }
+    case Provider::kTRT: {
+      struct TrtPairs {
+        const char* op_keys;
+        const char* op_values;
+      };
+
+      std::vector<TrtPairs> trt_options = {
+        {"device_id", "0"},
+        {"trt_max_workspace_size", "2147483648"},
+        {"trt_max_partition_iterations", "10"},
+        {"trt_min_subgraph_size", "5"},
+        {"trt_fp16_enable", "0"},
```

Comment:
I think we could enable fp16 here to get faster.

Reply:
Yess, I tried with fp16 also, and didn't see improvement in performance. 

But still with the config patch it will be an option for people to enable